libs/shared/src/components/UX/Numeric/styles.module.less

```diff
@@ -0,0 +1,43 @@
+@import "~@thecointech/site-semantic-theme/variables.less";
+
+.numeric {
+  :global(.ui.input)>input {
+    text-align: right;
```

Comment:
I'm not 100% convinced it is a good idea to enforce text-align here.  It looks much better, but it can be awkward to select